integ-test/src/test/java/org/opensearch/sql/calcite/standalone/CalciteBuiltinFunctionsNullIT.java

```diff
@@ -0,0 +1,1007 @@
+package org.opensearch.sql.calcite.standalone;
```

Comment:
1. copyright
2. rename to `CalcitePPLBuiltinFunctionsNullIT`
3. add `CalcitePPLBuiltinFunctionsNullPushdownIT`

Reply:
Done

Reply:
Done.